### PR TITLE
fix: add missing style.css file to initial download

### PR DESCRIPTION
**Pull request by sagewall**
The create a web app using components initial download is missing the `style.css` file.  This PR adds the missing blank file.

### CODE REVIEW

**Review by dasa — CHANGES_REQUESTED** [review]
new file is empty

File: component-samples/map-components/tutorials/create-a-web-app-using-components-initial/style.css
Comment (dasa) [inline component-samples/map-components/tutorials/create-a-web-app-using-components-initial/style.css:1]: Maybe a comment on why this is empty if that's intentional.

Comment (sagewall) [reply component-samples/map-components/tutorials/create-a-web-app-using-components-initial/style.css:1]: Sure, I agree it's a strange commit.  `style.css` is intended to be included in the initial download starter project as a blank file.  It's imported in starter `main.js` and leads to build errors and confusion where the tutorial assumes the empty file exists.  I guess we could consider editing the tutorial to create the file but this seems like the easier path forward for now is to add the empty `style.css` file

Comment (dasa) [reply component-samples/map-components/tutorials/create-a-web-app-using-components-initial/style.css:1]: maybe added a css comment to the file saying that styles should be added here

Comment (sagewall) [reply component-samples/map-components/tutorials/create-a-web-app-using-components-initial/style.css:1]: added 😃

**Review by dasa — APPROVED** [review]

**Pull request merged**